### PR TITLE
Update form_div_layout.html.twig - Remove CSS classes

### PR DESCRIPTION
**Pull request by seb-jean**
Hi,

I removed CSS classes in the file `form_div_layout.html.twig` because I think this file should not depend on any CSS framework.

What do you think?

### CODE REVIEW

File: src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig [hunk 1]
```diff
@@ -288,7 +288,7 @@
             {% set label_attr = label_attr|merge({'for': id}) %}
         {%- endif -%}
         {% if required -%}
-            {% set label_attr = label_attr|merge({'class': (label_attr.class|default('') ~ ' required')|trim}) %}
+            {% set label_attr = label_attr|merge({'class': (label_attr.class|default(''))|trim}) %}
```
Comment (stof) [inline src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig]: The whole line should be removed, as it does not modify the class anymore

File: src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig [hunk 2]
```diff
@@ -324,7 +324,7 @@
 
 {% block form_help -%}
     {%- if help is not empty -%}
-        {%- set help_attr = help_attr|merge({class: (help_attr.class|default('') ~ ' help-text')|trim}) -%}
+        {%- set help_attr = help_attr|merge({class: (help_attr.class|default(''))|trim}) -%}
```
Comment (stof) [inline src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig]: Same here about removing the line entirely

**Comment by jvasseur** [comment]
Even if I agree with the fact that symfony shouldn't add classes in its frameworkless form theme, this is a BC break for people relying on them (and it would be hard to provide an upgrade path with deprecations) so I don't think this is a good idea.

**Comment by YaFou** [comment]
> 
> 
> Even if I agree with the fact that symfony shouldn't add classes in its frameworkless form theme, this is a BC break for people relying on them (and it would be hard to provide an upgrade path with deprecations) so I don't think this is a good idea.

If we keep it like that, this PR should target the `6.0` branch and add an entry to the `UPGRADE` file.

**Comment by jvasseur** [comment]
Even if we target 6.0 we should add deprecation warnings in the 5.4 branch. But I don't know how we could trigger them properly since we can't know if they are used or not.

**Comment by xabbuh** [comment]
I agree with @jvasseur. So from my side that's a :-1:

**Comment by fabpot** [comment]
Closing for the reasons explained in the comments.

**Comment by Tobion** [comment]
The required class is a relict from a time when there was no required html attribute and no `:required` css selector.
I think it's time to remove it in Symfony 6.0.

**Pull request closed without merging**